WalletWasabi.Fluent/Controls/PrivacyContentControl.axaml

```diff
@@ -0,0 +1,61 @@
+<Styles xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:controls="using:WalletWasabi.Fluent.Controls"
+        xmlns:conv="clr-namespace:WalletWasabi.Fluent.Converters">
+  <Design.PreviewWith>
+    <controls:PrivacyContentControl />
+  </Design.PreviewWith>
+
+  <Style Selector="controls|PrivacyContentControl">
+    <Setter Property="Template">
+      <ControlTemplate>
+        <Panel Background="Transparent">
+          <ContentPresenter Name="PART_Content"
+                            Opacity="{TemplateBinding IsContentVisible, Converter={x:Static conv:BoolOpacityConverters.BoolToOpacity}}"
+                            IsVisible="{Binding $self.Opacity, Converter={x:Static conv:BoolOpacityConverters.OpacityToBool}}"
+                            ContentTemplate="{TemplateBinding ContentTemplate}"
+                            Content="{TemplateBinding Content}"
+                            VerticalContentAlignment="{TemplateBinding VerticalContentAlignment}"
+                            HorizontalContentAlignment="{TemplateBinding HorizontalContentAlignment}" />
+          <Panel Name="PART_PrivacyReplacement"
+                 Opacity="{TemplateBinding IsPrivacyContentVisible, Converter={x:Static conv:BoolOpacityConverters.BoolToOpacity}}"
+                 IsVisible="{Binding $self.Opacity, Converter={x:Static conv:BoolOpacityConverters.OpacityToBool}}">
+            <Viewbox Name="PART_Icon">
+              <PathIcon Data="{StaticResource incognito_regular}" Opacity="0.15" />
+            </Viewbox>
+            <TextBlock Name="PART_PrivacyText"
+                       IsVisible="{Binding !#PART_Icon.IsVisible}"
```

Comment:
Can IsVisible be also set for PART_PrivacyText from style similar to PART_Icon ?

Reply:
It is via style because it depends on `PrivacyReplacementMode`.
```
  <Style Selector="controls|PrivacyContentControl[PrivacyReplacementMode=Text] /template/ Viewbox#PART_Icon">
    <Setter Property="IsVisible" Value="False" />
  </Style>
  <Style Selector="controls|PrivacyContentControl[PrivacyReplacementMode=Icon] /template/ Viewbox#PART_Icon">
    <Setter Property="IsVisible" Value="True"/>
  </Style>
```

Reply:
I set both visibility via style to have the consistency